packages/eas-cli/src/build/runBuildAndSubmit.ts

```diff
@@ -138,9 +139,11 @@ export async function runBuildAndSubmitAsync(
       moreBuilds: platforms.length > 1,
       buildProfile,
       resourceClass:
-        platformToGraphQLResourceClassMapping[buildProfile.platform][
-          flags.userInputResourceClass ?? UserInputResourceClass.DEFAULT
-        ],
+        env.EXPO_USE_M1_RESOURCE_CLASS && buildProfile.platform === Platform.IOS
```

Comment:
Why are you checking envs? We are using resource classes so we can just use `--resource-class` param

Reply:
I used env to make this option more hidden for the users, although I'm open to adding it to `--resource-class` param